index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+
+    <title>Intuit Accounts - Sign Up</title>
+
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+
+    <header>
+        <div class="header-section">
+            <div class="signin-container">
+                <div class="signin-text">
+                    <p class="p-signin">Already have an Intuit account?</p>
+                    <a href="#">I forgot my User ID or Password</a>
+                </div>
+                <div>
+                    <button class="signin-button">
+                        Sign In
+                    </button>
+                </div>
+            </div>
+            <div class="logo-container center-div">
+                <a href="#"><img src="images/intuit-logo.png" alt="Intuit Logo"></a>
+            </div>
+            <div class="services-container center-div">
+                <a href="#" class="serviceIcon"><img src="images/turboIcon.png" alt="Turbotax Logo"></a>
+                <a href="#" class="serviceIcon"><img src="images/quickIcon.png" alt="Quickbooks Logo"></a>
+                <a href="#" class="serviceIcon"><img src="images/mintIcon.png" alt="Mint Logo"></a>
+            </div>
+        </div>
+    </header>
+
+    <main>
+
+        <section>
+            <form name="create-account">
```

Comment:
`AWESOME` 

Nice job using `form` element, this makes the section represent a document section that contains interactive controls for submitting information to a web server. :+1: